src/components/filter.js

```diff
@@ -78,21 +79,24 @@ const Filter = ({
             )
         }
       })}
-      <ListItem
-        innerDivStyle={style.listItemInner}
-        leftIcon={<SettingsIcon style={style.listIcon} />}
-        primaryText={<FormattedMessage {...messages.changeFilters} />}
+      <FlatButton
+        label={intl.formatMessage(messages.changeFilters)}
+        icon={<SettingsIcon style={style.listIcon} />}
+        onTouchTap={onOpenFilterConfigurator}
       />
     </List>
   </div>
 )
 
 Filter.propTypes = {
+  configureFilters: PropTypes.bool,
   filters: PropTypes.object,
   fieldStats: PropTypes.object.isRequired,
+  intl: intlShape.isRequired,
   visibleFilters: PropTypes.arrayOf(PropTypes.string).isRequired,
+  onOpenFilterConfigurator: PropTypes.func,
   /* called with valid mapbox-gl filter when updated */
   onUpdateFilter: PropTypes.func
 }
 
-module.exports = pure(Filter)
+module.exports = pure(injectIntl(Filter))
```

Comment:
This may conflict/complement `intlReducer`; I'm about to go read the docs about that.